src/main/java/core/basesyntax/ReportGenerator.java

```diff
@@ -0,0 +1,3 @@
+public interface ReportGenerator {
```

Comment:
The interface definition should not start with an empty line. Please remove the empty line at the beginning of the interface declaration.